src/CachePlugin.php

```diff
@@ -158,20 +158,20 @@ private function createCacheKey(RequestInterface $request)
      *
      * @param ResponseInterface $response
      *
-     * @return int|null
+     * @return int
      */
     private function getMaxAge(ResponseInterface $response)
     {
         if (!$this->config['respect_cache_headers']) {
-            return $this->config['default_ttl'];
+            return (int) $this->config['default_ttl'];
```

Comment:
Since the default value is null, this will become 0. Is that a good idea? Also, we have OptionsResolver in the plugin. @Nyholm any ide why null is allowed here?